api/client/trust.go

```diff
@@ -176,11 +176,16 @@ func convertTarget(t client.Target) (target, error) {
 }
 
 func (cli *DockerCli) getPassphraseRetriever() passphrase.Retriever {
-	baseRetriever := passphrase.PromptRetrieverWithInOut(cli.in, cli.out)
+	aliasMap := map[string]string{
```

Comment:
UX change: allows overriding the name of the keys that get presented to the users.